ubiconfig/_impl/loaders/gitlab.py

```diff
@@ -35,15 +37,21 @@ def load(self, file_name):
             raise
 
         # validate input data
-        validate_config(config_dict)
+        try:
+            validate_config(config_dict)
+        except ValidationError as e:
+            LOG.error("%s FAILED schema validation:\n%s\nSkip for now", file_name, e)
+            return
 
         return UbiConfig.load_from_dict(config_dict, file_name)
 
     def load_all(self, recursive=False):
         ubi_configs = []
         for file in self._files_branch_map:
             LOG.debug("Now loading %s from branch %s", file, self._files_branch_map[file])
-            ubi_configs.append(self.load(file))
+            config = self.load(file)
```

Comment:
Load can fail with YAMLError here, it will break execution, is that what you want?